src/storage/slab/slab.c

```diff
@@ -25,6 +25,11 @@ struct slab_heapinfo {
     struct slab_tqh slab_lruq;   /* lru slab q */
 };
 
+struct slab_pool_metadata {
+    void * usr_pool_addr;                 /* pool adresss */
```

Comment:
pointers next to variable names not types.

Reply:
Done.